src/EPR.CommonDataService.Api/Controllers/SubmissionsController.cs

```diff
@@ -76,7 +77,7 @@ public async Task<IActionResult> GetApprovedSubmissionsWithAggregatedPomData(str
             {
                 logger.LogError("{LogPrefix}: SubmissionsController - GetApprovedSubmissionsWithAggregatedPomData: The datetime provided did not return any submissions", _logPrefix);
                 ModelState.AddModelError(nameof(approvedAfterDateString), "The datetime provided did not return any submissions");
```

Comment:
In this case ModelState.AddModelError(...) not required any more because it's not used in this condition?

Reply:
Updated